### ISSUE TITLE
Crash related to Endermen

### ISSUE THREAD

**Issue by macf2802** [PixitOwO/Tinted#1]
I got a crash using the mod. As soon as I joined the world my game crashed until I removed Tinted from the mods folder. the mclogs link contains all information. If you need any other info let me know.
https://mclo.gs/FDqLO2E

**Comment by pixitshark** [comment]
Yeah I had issues with tinted in modpacks, it's something I will have to add a compat for. It seems that modifying the base textures and way flowers work can cause issues with other things like enderman holding flowers or mods with flowers involved.
